colossalai/fx/passes/algorithms/utils.py

```diff
@@ -0,0 +1,229 @@
+class Chain:
+
+    def __init__(self, fw, bw, cw, cbw, ftmp, btmp, check=True):
+        self.fweigth = fw
```

Comment:
`fweigth` not `fweight`? Is that typo from rotor?

Reply:
I will fix this and the above funny variable names